scripts/mdtoc.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Maybe we should put this script under `/tools` or `/test` to align with Kubeflow pipeline?

Reply:
good idea, I moved it to tools